app/src/main/res/values-sr/strings.xml

```diff
@@ -0,0 +1,160 @@
+    <string name="homescreen_handshakes_module_info_label_nosymptoms">Ovde ćete biti obavešteni kada neko od vaših susreta bude pozitivno testiran na COVID-19.</string>
+    <string name="inform_text_homescreen_nosymptoms">Obavesti ljude koje si sreo da si pozitivno testiran na COVID-19.</string>
+    <string name="unexpected_error_title">desila se nepoznata greška</string>
+    <string name="bluetooth_turned_off">Aplikacija ne radi, molim Vas aktivirajte Bluetooth.</string>
+    <string name="bluetooth_permission_turned_off">Aplikacija ne radi, Bluetooth autorizacija je isključena.</string>
+    <string name="NSBluetoothPeripheralUsageDescription">Susreti su prepoznati preko Bluetootha, radeći u pozadini štedeći energiju</string>
+    <string name="NSBluetoothAlwaysUsageDescription">Susreti su prepoznati preko Bluetootha, radeći u pozadini štedeći energiju</string>
+    <string name="foreground_service_notification_text">Aplikacija skenira u pozadini</string>
+    <string name="exposed_info_support_text">"Ukoliko imate bilo kakva pitanja ili trebate preporuke, pozovite info broj "</string>
+    <string name="inform_code_title">Unesite kod</string>
+    <string name="exposed_info_tel_button_title">19 819</string>
+    <string name="inform_code_text">Unesite kod koji ste primili sa test rezultatom</string>
+    <string name="inform_code_no_code">Niste dobili kod ?</string>
+    <string name="logs_filter_tags">tag filter (odvojen zarezom)</string>
+    <string name="bluetooth_setting_title">Slanje podataka</string>
+    <string name="bluetooth_setting_text">Omogući Bluetooth</string>
+    <string name="bluetooth_setting_tracking_active">Slanje podataka omogućeno</string>
+    <string name="bluetooth_setting_tracking_active_subtext">Susreti su sačuvani anonimno na tvom uređaju.</string>
+    <string name="bluetooth_setting_tracking_inactive">Slanje podataka zaustavljeno.</string>
+    <string name="bluetooth_setting_tracking_inactive_subtext">Omogući Bluetooth tako da susreti mogu biti sačuvani anonimno.</string>
+    <string name="bluetooth_setting_tracking_explanation_title">Kako slanje podataka radi ?</string>
+    <string name="bluetooth_setting_data_explanation_title">Da li su moji podaci sigurni ?</string>
+    <string name="bluetooth_setting_tracking_explanation_text1">"Preko BLE (Bluetooth Low Energy) aplikacija prepoznaje susrete sa drugim korisnicima. Takvi susreti će biti registrovani od strane aplikacije, ukoliko korisnici stoje blizu jedno drugog određeni vremenski period, i obrisani su posle 21 dan. "</string>
+    <string name="bluetooth_setting_tracking_explanation_text2">Susreti sa drugim korisnicima su sačuvani samo na vašem uređaju.</string>
+    <string name="bluetooth_setting_data_explanation_text1">Svaka aplikacija ima svoj dinamički ID koji je nasumično generisan preko «HMAC(SHA256)» i menja se svakih 15 minuta. Na ovaj način svaki korisnik ostaje anoniman.</string>
```

Comment:
Svaka aplikacija ima svoj dinamički ID koji se nasumično generiše pomoću «HMAC(SHA256)» i menja se svakih 15 minuta. Na ovaj način svaki korisnik ostaje anoniman.